Makefile

```diff
@@ -11,7 +11,10 @@ clean: ## Remove the build folder and the shared library
 	rm -rf build/ pyctbgui/_decoder.cpython*
 
 test: ## Run unit tests using pytest
-	python -m pytest 
+	python -m pytest --ruff -v
```

Comment:
I think I would split testing and linting in different actions. You could have 

```
make format # formats the code
make check_format #check only pass/fail
make test #only pytest tests
```

Since for me linting and testing are conceptionally different.

Reply:
yes makes sense